src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -264,9 +264,42 @@ private function fetchUrl(string $url, string $locale = null, int $limit, string
             if (isset($result->formatted_address)) {
                 $address = $address->withFormattedAddress($result->formatted_address);
             }
+            if ($builder->hasValue('streetAddress')) {
```

Comment:
@Nyholm  sorry for late response. I was thinking about replacing all this $builder->hasValue(..) with 
			if (!$builder->isEmptyDataArray())
			{
				$data = $builder->getDataArray();
				{
					foreach($data as $key => $value)
					{
						$address = $address->withData($key, $value);
					}
				}
			}


and 
public function withData(string $name, $value)
{
	$new = clone $this;
	$new->$name = $value;

	return $new;
}

Thats why I used camelCase in $data array. Should I do it, so we can discuss it?